parse.go

```diff
@@ -9,17 +9,25 @@ import (
 	"github.com/moby/buildkit/frontend/dockerfile/parser"
 )
 
+// Represents info about a heredoc.
+type Heredoc struct {
+	Name           string
+	FileDescriptor uint
```

Comment:
what is this value for? it doesn't seem used / useful

Reply:
Modified one of the tests to show the file descriptor. It is a heredocs funcitonality not used that often, but enough that it is handled by buildkit.